internal/schemautil/service.go

```diff
@@ -742,43 +745,147 @@ func copyConnectionInfoFromAPIResponseToTerraform(
 ) error {
 	props := make(map[string]interface{})
 
+	if len(connectionInfo.Direct) > 0 {
+		props["direct"] = connectionInfo.Direct
+	}
+
+	if connectionInfo.LetsEncrypt {
+		props["letsencrypt"] = connectionInfo.LetsEncrypt
+	}
```

Comment:
1. it won't store `false` value or ever go from `true` to `false` (if that is possible).
2. these two fields are missing from schemas

Reply:
Fair enough. I'm afraid, though, the current implementation doesn't allows this with other fields, too, for example strings, and other types.

Reply:
I think I will remove those fields for now because more work needs to be done to figure out which services actually use them.